### PR TITLE
Update LICENSE validity

### PR DESCRIPTION
**Pull request by SuperHarmony910**
This is a resolve to an orphaned PR (#6296)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Microsoft Corporation
+Copyright (c) present Microsoft Corporation
```
Comment (faxu) [inline LICENSE:3]: please remove "present" and it should be sufficient. Thanks!

**Review by faxu — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (SuperHarmony910) [reply LICENSE:3]: Is the reason [here](https://stackoverflow.com/a/2391555/1658314)

**Comment by SuperHarmony910** [comment]
The license has been updated to include the original year!

**Comment by SuperHarmony910** [comment]
Will this PR be merged? This change relates to [here](https://stackoverflow.com/a/2391555/1658314)

**Comment by snnn** [comment]
But, you didn't change any file.  There is nothing to merge.

**Comment by SuperHarmony910** [comment]
Very humiliating to have to confess that this PR meant nothing.

**Pull request closed without merging**